### PR TITLE
fix buttons on top

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-FEATURE] fix save & back-to-list buttons on top

### CODE REVIEW

File: BackofficeBundle/Resources/public/less/editTable.less [hunk 1]
```diff
@@ -1,7 +1,54 @@
-.back-to-list {
-    margin-bottom: 20px;
+#widget-grid .submit_form,
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/less/editTable.less]: You should use the less syntax that encapsulate children in their parent. It's one of the best feature of less :

    #widget-grid {
        .submit_form, .submit_form:active, .back-to-list, [...] {
            width: 119px;
        }
        .back-to-list {
            position: fixed;
            top: 54px;
            [....]
        }
    }

File: BackofficeBundle/Resources/public/less/editTable.less [hunk 2]
```diff
@@ -1,7 +1,54 @@
...
+#widget-grid .submit_form:active,
+#widget-grid .back-to-list,
+#widget-grid .back-to-list:active,
+#widget-grid .bh-button-add,
+#widget-grid .bh-button-add:active{
+    width: 119px;
+}
+#widget-grid .back-to-list {
+    /*margin-bottom: 20px;*/
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/less/editTable.less]: remove dead code

**Comment by djamnazi** [comment]
@itkg-ngilain PR updated

File: BackofficeBundle/Resources/public/less/editTable.less [hunk 3]
```diff
@@ -1,7 +1,40 @@
...
+        position: fixed;
+        top: 54px !important;
+        z-index: 901;
+    }
+    .back-to-list {
+        right: 15%;
+
+    }
+    .back-to-list:active, .submit_form:active, .bh-button-add:active {
+        left: 75.7% !important;
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/less/editTable.less]: I'm not sure about this kind of size: 75,7% seems to be very linked to your personnnal screen resolution. Have you tried with some other resolutions ? Have you tried with the fullscreen widget ?

File: BackofficeBundle/Resources/public/less/editTable.less [hunk 4]
```diff
@@ -1,7 +1,40 @@
...
+    .submit_form:active, .bh-button-add:active {
+        left: 85.7% !important;
+    }
+    .submit_form, .bh-button-add{
+        right: 5% !important;
+    }
+    .modal-upload {
+        .back-to-list {
+            top: 17px !important;
+            right: 5% !important;
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/less/editTable.less]: Again a mix of pixels and % generally lead to something broken on other resolutions than the one you used to developped.

**Pull request closed without merging**